apps/backoffice/public/locales/it/common.json

```diff
@@ -332,6 +332,20 @@
         "description": "Utilizza questa chiave per operazioni programmatiche di creazione di un nuovo servizio o aggiornamento di un servizio esistente.",
         "shortcut": "Gestisci API Key"
       },
+      "new-group-api-key": {
+        "title": "API Key dei Gruppi",
+        "description": "Seleziona il gruppo per il quale vuoi generare un API Key di gruppo.",
+        "card": {
```

Comment:
Consider to remove "card" level and flattening it

Reply:
[fix](https://github.com/pagopa/io-services-cms/pull/1047/commits/1db82e3f28ad9a3dabb402099f2f006d3175c2e3)